### PR TITLE
feat(input): added isClearable prop to input base

### PR DESCRIPTION
**Pull request by jamiesmyln**
re #266

### CODE REVIEW

**Review by orion-cengage — CHANGES_REQUESTED** [review]
We need to make sure the clear button styling is consistent across all the inputs.  The Combobox currently used the round button:
image

While the Search input is using the square button:
image

Since the Date Picker and Search inputs already use square buttons for their main action, let's just use square buttons for the Clear button as well.  This means updating Combobox.

File: packages/react-magma-dom/src/components/InputBase/index.tsx [hunk 1]
```diff
@@ -387,6 +431,28 @@ export const InputBase = React.forwardRef<HTMLInputElement, InputBaseProps>(
             />
           </IconButtonContainer>
         )}
+
+        {isClearable && value && (
+          <IsClearableContainer theme={theme}>
```
Comment (joasmi723) [inline packages/react-magma-dom/src/components/InputBase/index.tsx:436]: When I tab through the Search field, I notice the Search button gets focused before the X button, which doesn't match what I'm seeing visually.  Can we switch the order of these elements in the dom so it matches the visual?

File: packages/react-magma-dom/src/components/InputBase/index.tsx [hunk 2]
```diff
@@ -387,6 +431,28 @@ export const InputBase = React.forwardRef<HTMLInputElement, InputBaseProps>(
...
+            <IconButton
+              aria-label={'Clear Input'}
```
Comment (joasmi723) [inline packages/react-magma-dom/src/components/InputBase/index.tsx:438]: We should move this message into the i18n config.

**Review by joasmi723 — CHANGES_REQUESTED** [review]

File: packages/react-magma-dom/src/components/InputBase/index.tsx (hunk 1 above)
Comment (joasmi723) [reply packages/react-magma-dom/src/components/InputBase/index.tsx:428]: Awesome, this looks great!  

One other thing I hadn't noticed before is that after the clicking the 'clear' icon, nothing is in focus. I think we should return the focus to the search box.  

This PR also needs a changeset.  Everything else looks good to me.

**Pull request merged**